app/src/main/java/org/openobservatory/ooniprobe/activity/MeasurementDetailActivity.java

```diff
@@ -68,7 +71,7 @@
     Toolbar toolbar;
     private Measurement measurement;
     private Snackbar snackbar;
-
+    private Boolean isInExplorer;
```

Comment:
Please keep an empty line between private variables and methods. I find it more readable.